smoke/OK_readme-streams.pp

```diff
@@ -1,24 +1,23 @@
 #
-class {'riemann':
-  config_dir => '/tmp/riemann',
+class { 'riemann':
```

Comment:
yeah, like this is right `class { 'foo': }` but *this* is wrong `[ 'foo' ]` ... :trollface:

Reply:
Meh, same as Ruby, so quite used to this convention :shrug:

Caused by:

```
ERROR: there should be a single space or single newline after an opening brace on line 2 (check: manifest_whitespace_opening_brace_after)
```